Mes 3853 delete test results

richakl2 opened the pull request:
# Description and relevant Jira numbers

## Pull Request checklist

- [ ] [WIP] tag removed from PR title
- [ ] PR has an understandable description

## Git feature branch checklist

- [ ] branch name comply with our branching strategy
- [ ] git branch contains relevant JIRA ticket number
- [ ] branch rebased against the latest develop

## Sign off process checklist

- [ ] Code has been tested manually
- [ ] Tested by QA
- [ ] PO's approval
- [ ] Reviewers selected in Github
- [ ] PR link added to JIRA

richakl2 committed:
Removed some linting comments following Andy Review

richakl2 committed:
Removed some linting comments following Andy Review

richakl2 committed:
renamed deleteTestResultRecord function to deleteTestResult

richakl2 committed:
renamed deleteTestResultRecord function to deleteTestResult

richakl2 committed:
Merge branch 'develop' of https://github.com/dvsa/mes-result-service into mes-3853-delete-test-results

richakl2 committed:
Debugging DEV error

richakl2 committed:
Debugging DEV error

richakl2 committed:
Reverting to previous code

richakl2 committed:
Reverting to previous code

richakl2 committed:
Added code to also delete from the UPLOAD_QUEUE table when deleting TEST_RESULT

richakl2 committed:
Added code to also delete from the UPLOAD_QUEUE table when deleting TEST_RESULT

richakl2 committed:
Added code to also delete from the UPLOAD_QUEUE table when deleting TEST_RESULT

richakl2 committed:
Added code to also delete from the UPLOAD_QUEUE table when deleting TEST_RESULT

richakl2 committed:
Removed log of database connection used for debugging

richakl2 committed:
Merge branch 'mes-3853-delete-test-results' of https://github.com/dvsa/mes-result-service into mes-3853-delete-test-results

richakl2 committed:
Removed log of database connection used for debugging

andrewsetterfield commented on src/common/framework/mysql/database.ts:
```diff
@@ -1,9 +1,11 @@
 
 export const getConnection = (): mysql.Connection => {
   const configuration = config();
+  warn('config', configuration);
```
please remove

andrewsetterfield reviewed (changes requested):
please remove warning for config

richakl2 committed:
Removed log of database connection used for debugging

richakl2 committed:
Removed log of database connection used for debugging

richakl2 committed:
Removed log of database connection used for debugging

Removed log of database connection used for debugging

Removed log of database connection used for debugging

richakl2 committed:
Merge branch 'mes-3853-delete-test-results' of https://github.com/dvsa/mes-result-service into mes-3853-delete-test-results

richakl2 committed:
Removed log of database response used for debugging

richakl2 committed:
Removed log of database response used for debugging

richakl2 commented:
This is related to https://jira.dvsacloud.uk/browse/MES-3853
